bosh-windows-acceptance-tests/.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "assets/bwats-release/src/Pester"]
+	path = assets/bwats-release/src/Pester
+	url = https://github.com/pester/Pester.git
```

Comment:
Any idea why this is here? Do the tests use Pester?

Reply:
Yes, it does seem to be including Pester source files as a package in the `bwats-release`. Then runs pester tests: https://github.com/cloudfoundry/bosh-windows-acceptance-tests/blob/master/assets/bwats-release/jobs/check-system/templates/run.ps1.erb#L474-L480

So we do need this submodule, but this file needs to be moved to top-level of this repo.